src/firefly/js/charts/ui/XYPlotOptions.jsx

```diff
@@ -109,16 +109,36 @@ export function resultsSuccess(callback, flds, optionParameters) {
         stretch : flds.stretch,
         nbins : (nbinsX && nbinsY) ? {x: Number(nbinsX), y: Number(nbinsY)} : undefined,
         shading: flds.shading || undefined,
-        x : { columnOrExpr : xName, error: xErr, label : xLabel, unit : xUnit, options : xOptions},
-        y : { columnOrExpr : yName, error: yErr, label : yLabel, unit : yUnit, options : yOptions},
+        x : { columnOrExpr : xName, label : xLabel, unit : xUnit, options : xOptions},
+        y : { columnOrExpr : yName, label : yLabel, unit : yUnit, options : yOptions},
         tblId,
         zoom
     }, isUndefined);
 
-    if (xErr || yErr) {
-        if (xErr) { xyPlotParams.x.error = xErr; }
-        if (yErr) { xyPlotParams.y.error = yErr; }
-    }
+    // set error related parameters
+    ['x', 'y'].forEach((axis) => {
+        const errorsType = get(flds, [`${axis}.errorsType`]);
```

Comment:
get() take an array of paths or a string of path.
nothing is wrong with your statement.
but alternatively, you can do 
```
get(flds, `${axis}.errorsType`)  or
get(flds, [axis, 'errorsType'])
```